README.md

```diff
@@ -18,6 +18,8 @@ This repository enables Swift Package Manager support for the Fairmatic SDK for
 
 5. Click **Finish**
 
+6. Add the `-ObjC` flag to the *Other Linker Flags* section of your target's build settings.
```

Comment:
what about the `-lC++` flag, we don't need to add it explicitly?

Reply:
That is not needed as we are already linking the `c++` library using `.linkedLibrary("c++")`.